### PR TITLE
Feature/tp 11798 stepchange update

### PR DESCRIPTION
**Pull request by kiggundu**
This change delivers [TP-11798](https://maps.tpondemand.com/entity/11807-money-navigator-important-update-to-urgent) as part of the stepchange updates

### CODE REVIEW

**Comment by jordanlg27** [comment]
https://maps.tpondemand.com/entity/11798

File: app/models/concerns/money_navigator/rules/urgent_action.rb [hunk 1]
```diff
@@ -9,41 +9,41 @@ def self.all
...
-          # Any of these Q4A1, Q6A6, Q7A1-A9, Q10A3 PLUS the regional variation
+          # Any of these Q4A1, Q4A2, Q6A4, Q6A5, Q6A6, Q7A1-A9  BUT NOT IF HAVE SELECTED Q10A1 PLUS the regional variation
           heading_code: 'H1',
           content_rules: [
             {
               triggers: [
                 { q0: 'a1' },
-                { q4: 'a1', q6: ['a6'], q7: %w[a1 a2 a3 a4 a5 a6 a7 a8 a9], q10: 'a3' },
-                { q1: 'a2' }
+                { q4: %w[a1 a2], q6: %[a4 a5 a6], q7: %w[a1 a2 a3 a4 a5 a6 a7 a8 a9]},
```
Comment (davidtrussler) [inline app/models/concerns/money_navigator/rules/urgent_action.rb:18]: I think this should be `%w[a4 a5 a6]` (or commas as array separators). And likewise in the three instances below.

Comment (kiggundu) [reply app/models/concerns/money_navigator/rules/urgent_action.rb:18]: Ahhh.... good spot... looks like it might have come from a merge... 
not sure how the tests missed that as well... looking into it

Comment (davidtrussler) [reply app/models/concerns/money_navigator/rules/urgent_action.rb:18]: > Ahhh.... good spot... looks like it might have come from a merge...
> not sure how the tests missed that as well... looking into it
> 
Yeah I ran the tests with that as it is and changed to `%w` and it didn't make any difference, which is a bit odd.

Comment (kiggundu) [reply app/models/concerns/money_navigator/rules/urgent_action.rb]: The tests rightly passed because that rule contains q4 and q7 answers that were being provided. Reworking the testing to cover such an edge case would be quite involving (i.e. another 2 days of effort at least) so i've chosen to simply fix the mistake.

Comment (davidtrussler) [reply app/models/concerns/money_navigator/rules/urgent_action.rb]: Yeah, that seems fair enough!

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍
